platform-validation/scripts/aziot-compatibility.sh

```diff
@@ -792,16 +800,17 @@ check_free_memory() {
     current_free_memory=$(cat $memory_filename | grep "MemAvailable" | awk '{print $2/1024}')
 
     #TODO: correct final link of aka.ms/iotedge with the setup info of memory analysis.
-    base_message="IoT Edge requires a minimum memory of approximately $total_iotedge_memory_size MB for running the default setup as described in aka.ms/iotedge. We verified that the the device has $current_free_memory MB of free memory"
+    wrap_debug_message "IoT Edge requires a minimum memory of $total_iotedge_memory_size MB for a simple workload"
```

Comment:
Same comment as above. Also, we should not use terms like "simple workload".  Instead we can say something like " IoT Edge requires  approximately XX MB for running IoT Edge Binaries and Container Engine". For more information on how the memory profiling is done, please visit <insert-aka-ms-link>. We only display this message when --verbose flag is passed to the script, so we shouldnt worry about being as detail as possible